app/src/main/java/com/framgia/soundclound/data/source/remote/LoadAsync.java

```diff
@@ -0,0 +1,74 @@
+package com.framgia.soundclound.data.source.remote;
+
+import android.os.AsyncTask;
+
+import com.framgia.soundclound.data.source.TrackDataSource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.net.ssl.HttpsURLConnection;
+
+/**
+ * Created by Sony on 1/5/2018.
+ */
+
+public class LoadAsync extends AsyncTask<String, String, String> {
+    private TrackDataSource.Callback<String> mCallback;
+
+    public LoadAsync(TrackDataSource.Callback<String> callback) {
+        mCallback = callback;
+    }
+
+    @Override
+    protected void onPreExecute() {
+        mCallback.onStartLoading();
+    }
+
+    @Override
+    protected String doInBackground(String... strings) {
+        String urlStr = strings[0];
+        String result = null;
+        try {
+            URL url = new URL(urlStr);
+            HttpsURLConnection connection = (HttpsURLConnection) url.openConnection();
+            connection.setConnectTimeout(3000);
+            connection.setReadTimeout(3000);
+            connection.setRequestMethod("GET");
+            int responseCode = connection.getResponseCode();
+            if (responseCode != HttpURLConnection.HTTP_OK) {
+                throw new IOException("HTTP Code" + responseCode);
+            }
+            InputStream inputStream = connection.getInputStream();
+            result = readStream(inputStream);
+        } catch (IOException e) {
+            mCallback.onGetFailure(e.getMessage());
+        }
+        return result;
+    }
+
+    private String readStream(InputStream inputStream) throws IOException {
+        InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
```

Comment:
ERROR: [I18N][DM_DEFAULT_ENCODING] Found reliance on default encoding in com.framgia.soundclound.data.source.remote.LoadAsync.readStream(InputStream): new java.io.InputStreamReader(InputStream)
http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING